ports/mecab/portfile.cmake

```
@@ -10,7 +10,7 @@
 #   VCPKG_TARGET_ARCHITECTURE = target architecture (x64, x86, arm)
 #
 
-if (UWP OR ARM OR ARM64)
+if (TARGET_TRIPLET STREQUAL "x64-uwp" OR TARGET_TRIPLET STREQUAL "arm64-windows" OR TARGET_TRIPLET STREQUAL "arm-uwp")

```

I'm so sorry. I did a foolish mistake! Try to change this line to:

```suggestion
if (VCPKG_TARGET_TRIPLET STREQUAL "x64-uwp" OR VCPKG_TARGET_TRIPLET STREQUAL "arm64-windows" OR VCPKG_TARGET_TRIPLET STREQUAL "arm-uwp")
```

Sorry for my bad mistake. It should pass all the checks :)

---

changed this line, but still cannot pass all the checks :(